projects/fsastorefrontstyles/scss/components/cms/_enriched-responsive-banner.scss

```diff
@@ -2,9 +2,68 @@
   .enriched-banner {
     position: relative;
     width: 100%;
+    &-text {
```

Comment:
We should refactor this old code and stop breaking up selectors, as it's sometimes very hard to find their origin when declared like this. So just refactor form `&-text` to `.enriched-banner-text`. Please do this for the rest of the code as well.

Reply:
Then what's the point of scss if we don't you that syntax?

Reply:
The point is not just to create comfort for the developer who is writing the code, but also for the one that has to maintain it or even override it. On some projects it can happen that scss source is not on, and then you would get something like this:
 
<img width="452" alt="Screenshot 2021-08-12 at 13 22 42" src="https://user-images.githubusercontent.com/71004946/129189152-e6af47e4-d573-431f-8de9-a96fc8699c24.png">

With this you have no idea from where the style originates so you have to look for it with selector. Now, if that selector was written like above it would take you some time to find it because you would have to guess where someone has broken up the word for the selector. Of course that is before you start doubting that it may not be stored locally at all, but on some other server - remotely, because you cannot find `.enriched-banner-text`. So to save us all time and nerves I would suggest we start implementing this.

Reply:
So in this case there is no need to nest all those classes in `enriched-banner`? I can remove them outside of class `enriched-banner`?

Reply:
It would work as well if they stay nested because of the HTML structure, but you are right, it would be maybe better to move the outside, they would be easier then to override.